### PR TITLE
feat: bff 592 login

### PR DESCRIPTION
**Pull request by alexbejann**
## Type of Change

- [x] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: lib/core/network/api_service.dart [hunk 1]
```diff
@@ -0,0 +1,78 @@
+import 'dart:convert';
+
+import 'package:flutter/foundation.dart';
+import 'package:givt_app/core/network/interceptor.dart';
+import 'package:http/http.dart';
+import 'package:http_interceptor/http/http.dart';
+
+class APIService {
+  APIService() {
+    apiURL = kDebugMode ? 'givt-debug-api.azurewebsites.net' : 'api.givt.app';
```
Comment (MaikelStuivenberg) [inline lib/core/network/api_service.dart:10]: US has different url

File: lib/core/network/api_service.dart [hunk 2]
```diff
@@ -0,0 +1,78 @@
+import 'dart:convert';
+
+import 'package:flutter/foundation.dart';
+import 'package:givt_app/core/network/interceptor.dart';
+import 'package:http/http.dart';
+import 'package:http_interceptor/http/http.dart';
+
+class APIService {
```
Comment (MaikelStuivenberg) [inline lib/core/network/api_service.dart:8]: I thought you were going to use Dio for http?

File: lib/features/auth/pages/email_signup_page.dart [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+  @override
+  _InputPageState createState() => _InputPageState();
+}
+
+class _InputPageState extends State<InputPage> {
+  final _formKey = GlobalKey<FormState>();
+  final TextEditingController _emailController = TextEditingController();
+
+  @override
+  Widget build(BuildContext context) {
```
Comment (MaikelStuivenberg) [inline lib/features/auth/pages/email_signup_page.dart:36]: Wouldn't it be better if you split some of these long code blocks? Will make it more readable later if you need to lookup a specific text, button etc. 
This also applies to some other files.

**Comment by alexbejann** [comment]
Closing #1  as  #2 includes this changes.

**Pull request closed without merging**